### PR TITLE
add info on callout method in markdown

### PR DESCRIPTION
**Pull request by fakela**

### CODE REVIEW

File: content/writing/markdown.md [hunk 1]
```diff
@@ -61,3 +61,32 @@ will both render
 3. item 3
 
 The second way is more easy to manage when you have to reorder your list, but to ensure a better readability of the Markdown files when not rendered, we use the correct numbers for our list items.
+
+## Callout
+Callouts are a great method to attract additional attention to specific ideas or to make it obvious that some information is optional or only important in certain situations.
+
+Here is an example of how you to write a callout:
+
+```
```
Comment (fharper) [inline content/writing/markdown.md:70]: I would remove that since we list them we example rigth after

**Review by fharper — CHANGES_REQUESTED** [review]

**Comment by fharper** [comment]
Also... Callouts start with C so... 🤔

File: content/writing/markdown.md (hunk 1 above)
Comment (fakela) [reply content/writing/markdown.md:70]: I wanted to show an example of how it is written, most people wouldn't know the syntax used to achieve that

**Comment by fakela** [comment]
> Also... Callouts start with C so... 🤔

Umm, I used capital C

File: content/writing/markdown.md (hunk 1 above)
Comment (fharper) [reply content/writing/markdown.md]: We give them all examples right after

**Comment by fharper** [comment]
> > Also... Callouts start with C so... 🤔
> 
> Umm, I used capital C

They are not in alphabetic order within this file

File: content/writing/markdown.md (hunk 1 above)
Comment (fharper) [reply content/writing/markdown.md]: Oh my bad, they are visual!

**Review by fharper — APPROVED** [review]
LGTM

**Pull request merged**